src/components/TodoItem.js

```
@@ -0,0 +1,88 @@
+/* eslint-disable react/no-access-state-in-setstate */
+/* eslint-disable react/destructuring-assignment */
+/* eslint-disable react/state-in-constructor */
```

- [ ] Optional: Avoid disabling/ignoring errors.